tests/dslash_ctest.cpp

```diff
@@ -635,15 +635,7 @@ void dslashRef() {
         tm_dslash(spinorRef->V(), hostGauge, spinor->V(), inv_param.kappa, inv_param.mu, inv_param.twist_flavor, parity,
                   inv_param.matpc_type, dagger, inv_param.cpu_prec, gauge_param);
       else {
-        int tm_offset = 12 * spinorRef->Volume();
```

Comment:
@maddyscientist I wanted to ask if this is an acceptable change to the interface of the `host_reference` functions for the two-flavour operators. I've moved the offset calculation into the functions themselves.

Reply:
Yep, seems fine I think.

Note that I'll be merging in the develop branch into generic kernels in the next day or two, and that includes a signifcant cleanup of the dslash_test code to remove duplication between dslash_test and dslash_ctest.  So likely you'll have to deal with some conflicts.